errata_tool/cli/bug.py

```diff
@@ -0,0 +1,44 @@
+from errata_tool.bug import Bug
+
+
+def add_parser(subparsers):
+    """
+    Add build parser to this top-level subparsers object.
+    """
+    group = subparsers.add_parser('bug', help='Bugzilla information')
+
+    # build-level sub-commands:
+    sub = group.add_subparsers(dest='bug_subcommand')
+    sub.required = True
+
+    # "get"
+    get_parser = sub.add_parser('get')
+    get_parser.add_argument('id', help='bugzilla id of given '
+                            'bug')
+    get_parser.set_defaults(func=get)
+
+    get_all_parser = sub.add_parser('get_all_advisory_ids')
+    get_all_parser.add_argument('id', help='bugzilla id of given '
+                                'bug')
+    get_all_parser.set_defaults(func=get_all_advisory_ids)
+
+
+def get(args):
+    """
+    Get information about build
+
+    :param args: parsed cli arguments
+    """
+    build = Bug(args.id)
+    print(build)
+
+
+def get_all_advisory_ids(args):
```

Comment:
should we use the term `advisory` or `erratum` or `errata` ?

Reply:
As much as I wish we could do away with the terms `errata` and `erratum` and just use `advisory`, I see that in the rest of the code there are references to `errata_id` and none to `advisory_id`. So, for consistency (fewer ways to forget / be confused), I think new code should keep doing that.

I would also shorten this to the slightly less verbose
```suggestion
def get_errata_ids(args):
```